packages/lab/src/progress/LinearProgress/LinearProgress.css

```
@@ -104,9 +81,11 @@
 
 .saltLinearProgress-progressValue {
   color: inherit;
+  font-size: var(--salt-text-label-fontSize);
+  font-weight: var(--salt-text-label-fontWeight-strong);
   margin: 0;
   white-space: nowrap;
-  padding-left: var(--linearProgress-value-paddingLeft);
+  padding-left: var(--salt-size-unit);
```

```suggestion
  padding-left: var(--salt-spacing-100);
```
due to new spacing ramp :)